lib/KCard/index.vue

```diff
@@ -418,7 +422,7 @@
   /************* Layout-specific styles *************/
 
   .vertical-with-large-thumbnail {
-    height: 480px; /* (1) */
+    height: auto; /* (1) */
```

Comment:
Seems like this change is breaking the height of BelowTitle in vertical cards with large thumbnails
![image](https://github.com/user-attachments/assets/cc0065e1-a755-4c04-b0cc-ee81a5e9b606)

Reply:
Good catch, however it's fine in the scope of this PR. There's some limitations around heights that I am taking care of in https://github.com/learningequality/kolibri-design-system/pull/748 and this will to be resolved.